src/components/TracklistsCard/TracklistsCard.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
+import { Link } from 'gatsby'
 import styles from './TracklistsCard.module.scss'
 
-const TracklistsCard = ({ text, image, title, twitter, slack }) => (
-  <div className={styles.card}>
-    <img src={image} />
+const TracklistsCard = ({ url, text, image, title }) => (
+  <Link className={styles.card} to={url}>
+    <img alt={title} src={image} />
     {title && <p className={styles.title}>{title}</p>}
-    {text && <p>{text}</p>}
-  </div>
+    {text && <p className={styles.content}>{text}</p>}
```

Comment:
Use !!text as strings are valid React children as well and might get rendered wrong. Same goes for title above

Reply:
fixed